packages/splits-kit/src/constants/chains.ts

```diff
@@ -0,0 +1,638 @@
+import {
+  mainnet,
+  goerli,
+  polygon,
+  polygonMumbai,
+  optimismGoerli,
+  optimism,
+  arbitrumGoerli,
+  arbitrum,
+  gnosis,
+  fantom,
+  bsc,
+  avalanche,
+  aurora,
+  base,
+  zora,
+  zoraTestnet,
+  Chain,
+} from 'viem/chains'
+
+const SupportedChainsList = [
+  mainnet,
+  goerli,
+  polygon,
+  polygonMumbai,
+  optimism,
+  optimismGoerli,
+  arbitrum,
+  arbitrumGoerli,
+  gnosis,
+  fantom,
+  avalanche,
+  bsc,
+  aurora,
+  zora,
+  zoraTestnet,
+  base,
+]
+
+const DropdownDisplayChainsList = [
+  mainnet,
+  polygon,
+  optimism,
+  arbitrum,
+  zora,
+  base,
+  gnosis,
+  fantom,
+  bsc,
+  avalanche,
+  aurora,
+]
+
+export const SupportedChainsMap = SupportedChainsList.reduce(
+  (acc, chain) => {
+    acc[chain.id] = chain
+    return acc
+  },
+  {} as { [key: string]: Chain },
+)
+export const SupportedChains = SupportedChainsList
+export type SupportedChain = (typeof SupportedChains)[number]
+export type SupportedChainId = SupportedChain['id']
+
+export const SUPPORTED_CHAINS = SupportedChains
+export const DISPLAY_CHAINS = DropdownDisplayChainsList
+
+export const TESTNETS = [
+  goerli,
+  polygonMumbai,
+  optimismGoerli,
+  arbitrumGoerli,
+  zoraTestnet,
+]
+
+export enum SupportedNetwork {
+  ETHEREUM = 'ethereum',
+  POLYGON = 'polygon',
+  OPTIMISM = 'optimism',
+  ARBITRUM = 'arbitrum',
+  GNOSIS = 'xdai',
+  FANTOM = 'fantom',
+  AVALANCHE = 'avalanchec',
+  BSC = 'binance',
+  AURORA = 'aurora',
+  ZORA = 'zora',
+  BASE = 'base',
+}
+
+export interface L1ChainInfo {
+  readonly viemChain: Chain
+  readonly docs: string
+  readonly explorer: string
+  readonly explorerName: string
+  readonly network: SupportedNetwork
+  readonly label: string
+  readonly logoUrl: string
+  readonly rpcUrls: string[]
+  readonly websocketUrls: string[]
+  readonly nativeCurrency: {
+    name: string
+    symbol: string
+    decimals: number
+    logoUri: string
+    coingeckoId: string
+  }
+  readonly coingeckoErc20LookupId: string
+  readonly ensCoinType: number
+  readonly gqlUrl: string
+  readonly startBlock: number
+  readonly supportsEns: boolean
+  readonly supportsReverseEns: boolean
+  readonly openseaName?: string
+  readonly wrappedNativeTokenAddress?: string
+  readonly supportsSwapper?: boolean
+  readonly sponsorshipDisabled?: boolean
+  readonly distributionThresholdRange?: [number, number, number]
+  readonly gasPriceGwei?: number
+}
+
+export type ChainInfo = {
+  readonly [chainId in SupportedChainId]: L1ChainInfo
+}
+
+// merge with @usedapp & own utils getExplorer etc
+export const CHAIN_INFO: ChainInfo = {
```

Comment:
couple questions here:
- is this stuff used in the actual sdk? or just for storybook?
- and i think not all of the data is needed, right? like certainly the `supportsSwapper` field isn't needed. If this stuff is just used for storybook then less important. If it's being included in the sdk package though then I think trying to trim it down to what's necessary seems good.

Reply:
It's actually used in both, I will prune it down.